Arm/Insts/Common.lean

```diff
@@ -323,4 +319,24 @@ example : rev_vector 32 16 8 0xaabbccdd#32 (by decide)
           0xbbaaddcc#32 := by
           native_decide
 
+----------------------------------------------------------------------
+
+@[simp]
+def Elem_nassign (vector : BitVec n) (e : Nat) (size : Nat)
```

Comment:
Comments to describe these two functions would be helpful, given that they are in `Common.lean`. Also, may I suggest different names, like `elem_set` and `elem_get`?